account_financial_risk/models/res_partner.py

```diff
@@ -367,17 +367,17 @@ def _prepare_risk_account_vals(self, groups):
         # Partner receivable account determines if amount is in invoice field
         for (
             partner,
-            account,
+            _account,
             currency,  # noqa: B007
             amount_residual,
             amount_residual_currency,  # noqa: B007
         ) in groups["draft"]["read_group"]:
             if partner.id not in self.ids:
                 continue  # pragma: no cover
-            vals["risk_invoice_draft"] += account.company_id.currency_id._convert(
+            vals["risk_invoice_draft"] += currency._convert(
                 amount_residual,
                 self.risk_currency_id,
-                account.company_id,
+                self.env.company,
```

Comment:
I understand that `account.account` can now be shared across multiple companies, and the `company_id` field is now a `many2many`, according to [this commit](https://github.com/odoo/odoo/commit/854c3b27aa5476c208572f19e64f8f3364bfc381). So, I’m not sure if it’s the best approach to take the company from the environment.